src/jobflow_remote/config/helper.py

```diff
@@ -130,13 +130,20 @@ def _check_workdir(worker: WorkerBase, host: BaseHost) -> str | None:
         exc = traceback.format_exc()
         return f"Error while testing worker:\n {exc}"
 
+    canary_file = worker.work_dir / ".jf_heartbeat"
     try:
-        canary_file = worker.work_dir / ".jf_heartbeat"
+        # First try to create the folder. The runner will create is anyway and
+        # it should be less confusing for the user.
+        host.mkdir(worker.work_dir)
         host.write_text_file(canary_file, "\n")
```

Comment:
Do we need to check if the file exists first too in case of a bad cleanup? Or will `write_text_file` overwrite either way?

Reply:
Good point. I tested and it overwrites the file in both the local and remote worker. I think it can be leaved like this then.